app/views/pages/visit.html.erb

```diff
@@ -3,37 +3,15 @@
 <div class="container" style="margin-top: 35px;">
 
 	<div class="row">
-		<div class="col-12 col-md-6 col-lg-4 text-center">
-			<%= link_to (image_tag "book.png", class: "category-icon"), controller: "pages", action: "visit" %>
-			<h2><%= link_to "Borrowing", controller: "pages", action: "visit" %></h2>
-			<p>Lorem ipsum magillicus</p>
-		</div>
-		<div class="col-12 col-md-6 col-lg-4 text-center">
-			<%= link_to (image_tag "computer.png", class: "category-icon"), controller: "pages", action: "visit" %>
-			<h2><%= link_to "Computers, Printing, & Technology", controller: "pages", action: "visit" %></h2>
-			<p>Lorem ipsum magillicus</p>
-		</div>
-		<div class="col-12 col-md-6 col-lg-4 text-center">
-			<%= link_to (image_tag "study.png", class: "category-icon"), controller: "pages", action: "visit" %>
-			<h2><%= link_to "Study Spaces", controller: "pages", action: "visit" %></h2>
-			<p>Lorem ipsum magillicus</p>
-		</div>
+		<% @categories.each do |category| %>
+					
+					<div class="col-12 col-md-6 col-lg-4 text-center">
```

Comment:
In `app/views/pages/about.html.erb` you check for presence of a `category.icon`, but in this template you do not? Seems like we'd always want to check and fall back on a default.

If that is the case, please refactor rendering an individual category into a partial that can be used across all pages what have groups of categories.

Reply:
This page will be revised in a separate ticket. It's not part of the work done for this ticket as part of the redesign.

Reply:
This file actually should have no changes for this ticket. Just saw that it does, will revert, and add in the partial for displaying categories. Although, the contact us category will be different as we have to allow for the chat widget.